missileserver.py

```diff
@@ -96,6 +100,9 @@ def do_POST (self):
                 self.do_everything()
 
             def do_everything (self):
+		if "pass="+PASS not in self.path:
```

Comment:
tiny nitpick - the leading whitespace in the pull is tabs, whereas in the rest of the file it's spaces.  i'm sure you've tested it, but this is a 'feature' of python that can cause tough-to-find bugs.